homeassistant/components/wallbox/sensor.py

```diff
@@ -145,6 +146,11 @@ class WallboxSensorEntityDescription(SensorEntityDescription):
         device_class=SensorDeviceClass.CURRENT,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    CHARGER_OCPP_STATUS_KEY: WallboxSensorEntityDescription(
+        key=CHARGER_OCPP_STATUS_KEY,
+        translation_key=CHARGER_OCPP_STATUS_KEY,
+        icon="mdi:transit-connection-variant"
```

Comment:
icons are now handled using the translations logic (icon.json file). Can you change this to conform?